lib/PWC/Utils/DataMigration.pm

```diff
@@ -82,12 +75,12 @@ sub populate_member_folder {
 
             my $weekly_challenge = basename($weekly_challenge_folder);
             my $new_member_weekly_challenge_folder = "$member_folder/$weekly_challenge";
```

Comment:
Change the week folder from "challenge-123" to simply "w123". I think shorter week name is better.